modules/gitops-uninstall-keycloak.adoc

```diff
@@ -0,0 +1,23 @@
+[id="uninstalling-keycloak_{context}"]
+= Uninstalling Keycloak
+
+You can delete the Keycloak resources and its relevant configuration by removing the SSO field from Argo CD Custom Resource Spec. After removing the SSO field, the spec looks like the following:
+
+[source,yaml]
+----
+  apiVersion: argoproj.io/v1alpha1
+  kind: ArgoCD
+  metadata:
+    name: example-argocd
+    labels:
+      example: basic
+  spec:
+    server:
+      route:
+       enabled: true
+----
+
+[NOTE]
+====
+A Keycloak application created by this method is currently not persistent. In case of restarts, any additional configuration created in the Argo CD Keycloak realm is deleted.
```

Comment:
```suggestion
A Keycloak application created by using this method is currently not persistent. Additional configurations created in the Argo CD Keycloak realm are deleted when the [server????] restarts.
```
`In case of restarts` is not clear. What is being restarted?